pages/cloudflare/howtos/keep_names.mdx

```diff
@@ -0,0 +1,38 @@
+import { Callout } from "nextra/components";
+
+## `__name` issues
+
+When using the OpenNext adapter, Wrangler processes the worker's code with [esbuild](https://esbuild.github.io/), and by default
+enables the [`keep-names`](https://esbuild.github.io/api/#keep-names) option. While this is generally useful for debugging, it can
+cause issues with certain Next.js libraries (e.g. [next-themes](https://www.npmjs.com/package/next-themes)) that convert scripts
+to strings. This happens because `esbuild` introduces an `__name` function at the top of modules, which may inadvertently appear
```

Comment:
is that correct?
```suggestion
to strings. This happens because `esbuild` introduces a `__name` function at the top of modules, which may inadvertently appear
```